test/web-platform-tests/to-upstream/html/semantics/forms/the-input-element/input-stepup-realignment.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<title>Forms</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<h3>input_stepUp/Down realignment</h3>
+<input type=number id=inputNumber>
```

Comment:
Remove h3, change title to something informative, and remove test() description?

Reply:
Done.